apps/examples/testcase/Kconfig_ENTRY

```diff
@@ -1,3 +0,0 @@
-config ENTRY_TESTCASE
-	bool "TestCase Example"
-	depends on EXAMPLES_TESTCASE
```

Comment:
If you remove this, we can select the testcase using entry selection menu.

Reply:
Changes to this folder have been reverted